woocommerce/woocommerce
# Product Collection: move "Display Settings" from Toolbar to Inspector Controls

### Changes proposed in this Pull Request:





Closes #52622

Similar to PR https://github.com/WordPress/gutenberg/pull/58207 in Gutenberg repo, this PR moves the "Display Layout" menu from the block Toolbar to the Inspector Controls panel. This change improves the discoverability of display settings and aligns with the pattern used in core blocks.

The changes include:
- Moving the display layout controls from the block toolbar to the Inspector Controls panel
	- All of display settings are moved to "Settings" section
	- Keeping "Offset" and "Max pages to show" hidden under three dots menu
	- Placing "Products per page" under "Columns" control as they're related
- Removing the toolbar "Display Settings"
- Removing "Dimensions" section for now as discussed in Slack (https://a8c.slack.com/archives/C05UMTA7W0P)



### How to test the changes in this Pull Request:



**Case 1: Verify UI changes in Inspector Controls**
1. Create a new post and add a Product Collection block.
2. Click on "create your own" to add Custom Collection. 
3. Verify that "Products per page" is available in "Settings" section.
4. Verify that by default both "Offset" and "Max pages to show" are hidden i.e. not visible in Settings section.
5. Click on three dots menu and verify that "Offset" and "Max pages to show" available in dropdown menu.
6. Verify that you are able to add "Offset" and "Max pages to show" to Settings section by clicking on three dots menu and selecting these options.
<img width="280" alt="image" src="https://github.com/user-attachments/assets/10cb0770-afc5-44e4-9518-22f31f1d1927">

6. Verify that there is no "Dimensions" section in Inspector controls. 
	- "Width" control (Fill and Fixed) should be available in "Settings" section.

**Case 2: "Products Per Page" control"**
1. Change "Products Per Page" value, for example 2.
2. As soon as you change the value, it should update number of products on Editor side. In our case, 2 products should be shown.
3. Save the changes.
4. Verify on Frontend that number of products is 2.

**Case 3: "Offset" control**
1. Add "Offset" control by clicking on three dots and selecting "Offset".
2. Test offset functionality:
   - Set the "Offset" value to 2
   - Verify in the editor that:
     - The first 2 products are skipped
     - The product list starts from the 3rd product
   - The changes should be reflected immediately without needing to save

3. Verify frontend behavior:
   - Save the post
   - View the post on the frontend
   - Confirm that the offset is working correctly:
     - First 2 products are skipped
     - Product list starts from the 3rd product
     - The display matches what was shown in the editor

**Case 4: "Max Pages to Show" control**
1. Add "Max Pages to Show" control by clicking on three dots and selecting "Max Pages to Show"
2. Set "Products Per Page" to 2 to ensure multiple pages
3. Set "Max Pages to Show" to 2
3. Verify frontend behavior:
   - Save the post
   - View the post on the frontend
   - Confirm that:
     - Only 2 page numbers are displayed in pagination
     - You can still navigate between available pages




### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 
Product Collection: move "Display Settings" from Toolbar to Inspector Controls
</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
